src/ui/components/SecondaryToolbox/ReactDevTools.tsx

```diff
@@ -630,6 +632,13 @@ export function ReactDevtoolsPanel() {
     };
   }, [wall, store]);
 
+  useEffect(() => {
+    if (wall && currentPoint !== null) {
+      console.log("Starting RDT injection from effect");
+      wall.ensureReactDevtoolsBackendLoaded();
```

Comment:
This looks a little racy, because of the implementation of `ensureReactDevtoolsBackendLoaded`. I'll comment more there– but this effect will run multiple times so I think we'll need to make `ensureReactDevtoolsBackendLoaded` more robust to being called multiple times.